### ISSUE TITLE
FindLAPACK.cmake doesn't check the default LAPACK path on Ubuntu 20.04 x86_64

### ISSUE THREAD

**Issue by imaginary-person** [pytorch/pytorch#50996]
## 🐛 Bug

On Ubuntu 20.04 x86_64, by default, `apt-get` installs `LAPACK` in its corresponding subdirectory in `/usr/lib/x86_64-linux-gnu/`. 
But this path has not been included in `FindLAPACK.cmake`, so while building PyTorch from source, `setup.py` builds PyTorch without `LAPACK`.
Moreover, no environmental variable can be used to provide `LAPACK`'s path during installation,
so this issue might exist on several other linux distros as well.

## To Reproduce

Try building PyTorch from source on Ubuntu x86_64 if `apt-get` was used to install `LAPACK` without changing its default installation path.

## Expected behavior

`FindLAPACK.cmake` should check for the `LAPACK` installation directory in `/usr/lib/x86_64-linux-gnu/` on Linux as well,
so that `setup.py` would be able to build PyTorch with LAPACK support.

## Environment

**OS:** Ubuntu 20.04.1 LTS (x86_64)

cc @malfet @seemethere @walterddr

**Comment by walterddr** [comment]
which BLAS library you are using for this installation. according to https://github.com/pytorch/pytorch/blob/master/cmake/Modules/FindLAPACK.cmake#L12-L13. It is locating the LAPACK library that matches blas library

**Comment by imaginary-person** [comment]
Thanks for your response, and sorry for the inconvenience! 
Closing this issue, as I'm unable to reproduce it now.
`CMAKE_LIBRARY_PATH` also has  `/usr/lib/x86_64-linux-gnu/`, so `find_library()` should be able to locate BLAS & then LAPACK. My generic BLAS library is also at `/usr/lib/x86_64-linux-gnu/`.

I cleaned up the CMake cache & tried to rebuild by reverting the change in `FindLAPACK.cmake`, but the setup script was able to find both BLAS & LAPACK this time. 

However, with the previous build, tests related to LAPACK were being skipped due to PyTorch not having been compiled with LAPACK, so now I'm not sure why that had been happening, as running setup.py earlier showed it could not find BLAS and LAPACK. Then I installed both but setup.py couldn't find them until I changed `FindLAPACK.cmake`.

I'll reopen the issue if I would be able to figure out the correct reproduction steps.
